### PR TITLE
Rename rust steps

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: .github/workflows/ci-rust.yml [hunk 1]
```diff
@@ -91,15 +91,15 @@ jobs:
           RUST_LOG: debug
         timeout-minutes: 10
 
-      - name: Check non bindings crates using sodium crypto features
+      - name: Check platform crates using sodium crypto features
```
Comment (touilleMan) [inline .github/workflows/ci-rust.yml:94]: Agnostic is also tested here (hence we test everything except the bindings)

File: .github/workflows/ci-rust.yml [hunk 2]
```diff
@@ -91,15 +91,15 @@ jobs:
...
         run: cargo check ${{ env.CARGO_CI_FLAGS }} ${{ steps.crates.outcome.agnostic }} ${{ steps.crates.outcome.platform }} --features use-sodiumoxide
         timeout-minutes: 10
         env:
           RUST_LOG: debug
 
       # Use sodiumoxide here given 1) it is composed of C code, so not totally
       # platform independant and 2) it is what is going to be used in release
-      - name: Test Rust codebase (🐧 Linux specific)
+      - name: Test Platform Rust codebase using sodium crypto features (🐧 Linux specific)
```
Comment (touilleMan) [inline .github/workflows/ci-rust.yml:102]: "Test platform crates using ..." to have consistant wording?

**Review by touilleMan — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-rust.yml (hunk 1 above)
Comment (FirelightFlagboy) [reply .github/workflows/ci-rust.yml]: Done, I missed this one

File: .github/workflows/ci-rust.yml (hunk 2 above)
Comment (FirelightFlagboy) [reply .github/workflows/ci-rust.yml]: Done

**Review by touilleMan — APPROVED** [review]

**Pull request merged**